src/components/App/App.component.jsx

```diff
@@ -1,57 +1,24 @@
-import React, { useLayoutEffect } from 'react';
-import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import React from 'react';
+import styled from 'styled-components';
+import Header from '../Header/Header.component';
+import CardContainer from '../CardContainer/CardContainer.component';
 
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+const Main = styled.div`
+  display: flex;
+  justify-content: center;
+  padding: 4rem 10rem 3rem;
+  font-family: 'Roboto', sans-serif;
+  background-color: #191919;
+`;
```

Comment:
I recommend moving all the CSS from the components file, it is better if we keep separate business logic from the styling